ci/setup-build-test-gdk.ps1

```diff
@@ -6,6 +6,7 @@ param(
     [string] $unreal_engine_symlink_dir = "$build_home\UnrealEngine",
     [string] $gyms_version_path = "$gdk_home\UnrealGDKTestGymsVersion.txt"
 )
+. "$PSScriptRoot\common.ps1"
```

Comment:
not strictly moving this up meant no more silent failures causing the build to succeed but no tests to run.